docs/getting-started/python.md

```diff
@@ -9,6 +9,13 @@
 
 ### Install CDK for Terraform CLI
 
+Install with [Homebrew](https://brew.sh):
```

Comment:
Might be worth pointing out that this method is mac (I suppose or Linux) specific.